assignments/lambda-classes.js

```diff
@@ -1 +1,121 @@
 // CODE here for your Lambda Classes
+
+class Person {
+    constructor(attributes) {
+    this.name = attributes.name;
+    this.age = attributes.age;
+    this.location = attributes.location;
+    this.gender = attributes.gender;
+    }
+    speak() {
+        console.log(`Hello my name is ${this.name}, I am from ${this.location}.`);
+    }
+}
+
+class Instructor extends Person {
+    constructor(instructorAttributes) {
+        super(instructorAttributes)
+        this.specialty = instructorAttributes.specialty;
+        this.favLanguage = instructorAttributes.favLanguage;
+        this.catchPhrase = instructorAttributes.catchPhrase;
+    }
+
+    demo(subject) {
+        console.log(`Today we are learning about ${this.subject}.`); 
```

Comment:
this should just be subject instead of this.subject since the subject is being passed into the function and is not a property of instructor